app/scenes/channel_drawer/channel_drawer.js

```diff
@@ -27,6 +27,19 @@ export default class ChannelDrawer extends PureComponent {
         currentChannel: {}
     };
 
+    canPress = true;
+
+    handlePress = (action, ...args) => {
+        if (this.canPress) {
+            this.canPress = false;
+            action(...args);
```

Comment:
Does `Reflect.apply` need to be used here to make that the correct `this` is passed?